drivers/i2c/i2c_mspm0g3xxx.c

```diff
@@ -636,6 +636,8 @@ static int i2c_mspm0g3xxx_init(const struct device *dev)
 	DL_I2C_enablePower((I2C_Regs *)config->base);
 	delay_cycles(POWER_STARTUP_DELAY);
 
+	DL_I2C_disableTargetWakeup((I2C_Regs *)config->base);
```

Comment:
Please add a comment above this referring to the errata. Something like
`workaround for errata XYZ where target-wakeup can cause the target to hold SCL low indefinitely`
and maybe even a short note on the consequences of disabling target wakeup

Reply:
i will wait for TI's response on the issue so i can provide a more concrete comment/commit message